test/spec/EditorManager-test.js

```diff
@@ -85,14 +85,21 @@ define(function (require, exports, module) {
                 spyOn(pane, "showView");
                 spyOn(anotherPane, "addView");
                 spyOn(anotherPane, "showView");
-                var editor = SpecRunnerUtils.createEditorInstance(testDoc, anotherPane.$el, undefined, 'first-pane');
+
+                var editor = SpecRunnerUtils.createEditorInstance(testDoc, anotherPane, undefined);
                 EditorManager.openDocument(testDoc, anotherPane);
+
                 expect(pane.addView).not.toHaveBeenCalled();
                 expect(pane.showView).not.toHaveBeenCalled();
-                expect(anotherPane.addView).toHaveBeenCalled();
-                expect(anotherPane.addView).not.toHaveBeenCalledWith(editor);
-                expect(anotherPane.showView).toHaveBeenCalled();
-                expect(anotherPane.showView).not.toHaveBeenCalledWith(editor);
+                expect(anotherPane.addView).toHaveBeenCalledWith(editor);
+                expect(anotherPane.addView).toHaveBeenCalledWith(editor);
```

Comment:
I don't think this line is meant to be duplicated? (Same below with `expect(pane.showView).not.toHaveBeenCalledWith(editor);`)